inlang/source-code/paraglide/paraglide-js/example/tsconfig.json

```diff
@@ -8,6 +8,7 @@
 		"allowJs": true,
 		"checkJs": true,
 		"outDir": "dist",
+		"skipLibCheck": true,
```

Comment:
Is this a leftover or would such a configuration be necessary for all descendants like `sv` to also setup new projects like this?

Reply:
Just done to accelerate compile times on my machines. Does this change affect the output of `sv`? If so, I'll reverse it.

Reply:
Since this is only the example, it should not. Should be good to go!
I was worried that this might be a requirement, but this does not seem to be the case.